src/main/target/HAKRCF722/target.h

```diff
@@ -63,8 +63,15 @@
 #define USE_ACC_SPI_MPU6000
 #define ACC_1_ALIGN             CW270_DEG
 
+#define ICM20689_CS_PIN          PA4 
+#define ICM20689_SPI_INSTANCE    SPI1
 
-#define LED0_PIN                PC14
+#define USE_GYRO_SPI_ICM20689
```

Comment:
What gyro options will be sold for this board? Is a version with MPU6000 going to be available?

Reply:
Can only use one gyro sensor ?   We will delect the code of  ICM20689

Reply:
Having both gyros listed in the target definition is possible. I am just trying to make sure that the target definition is accurate.